web/src/lib/components/layouts/user-page-layout.svelte

```diff
@@ -40,3 +63,35 @@
     {/if}
   </slot>
 </main>
+{#if selectHidden}
```

Comment:
This is a full screen modal, basically, so you can move this to another component as well since it will render on top the all the other content on the screen.